### PR TITLE
[WebInterface] all exchange_type config

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/704
requires https://github.com/Drakkar-Software/OctoBot-Trading/pull/707

### CODE REVIEW

File: Backtesting/collectors/exchanges/exchange_bot_snapshot_data_collector/bot_snapshot_with_history_collector.py [hunk 1]
```diff
@@ -59,7 +59,7 @@ def __init__(self, config, exchange_name, tentacles_setup_config, symbols, time_
         self.set_file_path()
 
     def get_permanent_file_identifier(self):
-        symbols = "-".join(symbol_util.merge_symbol(symbol) for symbol in self.symbols)
+        symbols = "-".join(symbol_util.merge_symbol(symbol.symbol_str).replace(":", "_") for symbol in self.symbols)
```
Comment (Herklos) [inline Backtesting/collectors/exchanges/exchange_bot_snapshot_data_collector/bot_snapshot_with_history_collector.py:62]: I think we should put this conversion in symbol_util

File: Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py [hunk 2]
```diff
@@ -33,6 +33,7 @@
 
 class ExchangeHistoryDataCollector(collector.AbstractExchangeHistoryCollector):
     IMPORTER = generic_exchange_importer.GenericExchangeDataImporter
+    DEFAULT_START_TIMESTAMP = 631152000    # 01/01/1990
```
Comment (Herklos) [inline Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py:36]: 👍

File: Services/Interfaces/web_interface/models/backtesting.py [hunk 3]
```diff
@@ -359,10 +360,11 @@ async def _start_collect_and_notify(data_collector_instance):
 
 
 async def _background_collect_exchange_historical_data(exchange, symbols, time_frames, start_timestamp, end_timestamp):
+    symbols = symbols if isinstance(symbols, list) else [symbols]
```
Comment (Herklos) [inline Services/Interfaces/web_interface/models/backtesting.py:363]: How couldn't `symbols` be a list?

**Review by Herklos — APPROVED** [review]

File: Backtesting/collectors/exchanges/exchange_bot_snapshot_data_collector/bot_snapshot_with_history_collector.py (hunk 1 above)
Comment (GuillaumeDSM) [reply Backtesting/collectors/exchanges/exchange_bot_snapshot_data_collector/bot_snapshot_with_history_collector.py:62]: I'm not sure as this is a specific conversion to create the backtesting file name only, it's only used here

File: Services/Interfaces/web_interface/models/backtesting.py (hunk 3 above)
Comment (GuillaumeDSM) [reply Services/Interfaces/web_interface/models/backtesting.py:363]: I checked and I can't see any reason, I'll remove this line and see if it break things

**Review by Herklos — APPROVED** [review]

**Pull request merged**